virttest/utils_test/libvirt.py

```diff
@@ -3647,16 +3647,23 @@ def check_logfile(search_str, log_file, str_in_log=True,
                                           log_file))
 
 
-def check_qemu_cmd_line(content, err_ignore=False):
+def check_qemu_cmd_line(content, err_ignore=False,
+                        cmd_params=None, runner_on_target=None):
```

Comment:
Maybe define cmd_params as one dict, which will have more extension by wrapping  possible parameters in future.

Reply:
The params is already a dict type now.

Reply:
runner_on_target=None is not necessary now, could be wrapped into cmd_params